Tasks/GitHubPRComment/githubComment/index.ts

```diff
@@ -44,28 +55,61 @@ const iterateFilesFromDir = (
     return result;
 };
 
-const combineMessageBody = (files: string[]): string => {
-    var body: string = "";
-    files.forEach((file) => {
-        var bodyFile = fs.readFileSync(file);
-        body += bodyFile + "\r\n";
+const getFilesFromDir = (
+    filePath: string,
+    extName: string,
+    recursive: boolean
+): string[] => {
+    if (!fs.existsSync(filePath)) {
+        console.log(`File path does not exist: ${filePath}`);
+        return [];
+    }
+    var result: string[] = [];
+    var fileNames: string[] = [];
+    iterateFilesFromDir(filePath, extName, recursive, result);
+    result.forEach(element => {
+        var fileName = path.basename(element);
+        fileNames.push(fileName);
     });
-    return body;
+
+    taskLibrary.debug(`[iterateFilesFromDir] Files with extension ${extName} found: ` + fileNames);
```

Comment:
Shouldn't be getFilesFromDir?
```suggestion
    taskLibrary.debug(`[getFilesFromDir] Files with extension ${extName} found: ` + fileNames);
```